src/Command/TwoStepVerificationCommand.php

```diff
@@ -71,36 +59,15 @@ public function configure(): void
     /**
      * {@inheritdoc}
      */
-    public function execute(InputInterface $input, OutputInterface $output): void
+    public function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Since you're currently targeting `master`, you can add the proper return type declaration.
```suggestion
    public function execute(InputInterface $input, OutputInterface $output): int
```